tests/unit/test_bucket.py

```diff
@@ -3351,96 +3348,14 @@ def test_make_private_recursive_too_many(self):
 
         client.list_blobs.assert_called_once()
 
-    def test_page_empty_response(self):
```

Comment:
Why are these tests being removed?

Reply:
They are non-unit tests (they really only test the already-well-tested `api_core.page_iterator.Iterator`).  The local bits are replaced by the (new) explicit tests of `_item_to_blob` and `_blobs_page_start` (at the top of the file).